runtime/mrc/src/lib.rs

```diff
@@ -529,7 +534,9 @@ construct_runtime!(
 		// MRC related pallets
 		Msa: pallet_msa::{Pallet, Call, Storage, Event<T>} = 34,
 		Messages: pallet_messages::{Pallet, Call, Storage, Event<T>} = 35,
-		Schemas: pallet_schemas::{Pallet, Call, Storage, Event<T>} = 36,
+		Schemas: pallet_schemas::{Pallet, Call, Storage, Event<T>, Config} = 36,
+		Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T> }= 37,
```

Comment:
nit: sudo is not a MRC pallet

Reply:
I thought we already had this pallet.

Reply:
I updated this so you can disable a sudoer by passing rootkey param as None and it will work.